crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -246,6 +246,26 @@ jQuery(document).ready(function($) {
     },
     500
   );
+
+  $('[data-toggle="tooltip"]').tooltip({html : true});
+  $('[data-toggle="inline"]').popover({
+    html : true,
+    content: function() {
+      return $($(this).data('inline')).html();
+    }
+  });
+
+  $('body.backups input[name="file"]').fileinput({
+    uploadUrl: Routes.upload_backups_path({
```

Comment:
Identifier 'upload_backups_path' is not in camel case.<br>'Routes' is not defined.